build/assets/custom-dictionary.txt

```diff
@@ -254,6 +259,7 @@ Gottardo
 Governale
 GRCh
 Greiner
+grey
```

Comment:
gray

Reply:
this is an either or word with grey as my preferred spelling, but if you have a strong preference for this, we can switch - I won't make judgements (my preferred spelling of this one as well 😄 )

Reply:
Doesn't terribly matter to me!